pages/common/dd.md

```diff
@@ -3,26 +3,22 @@
 > Convert and copy a file.
 > More information: <https://www.gnu.org/software/coreutils/dd>.
 
-- Make a bootable USB drive from an isohybrid file (such like `archlinux-xxx.iso`):
+- Make a bootable USB drive from an isohybrid file (such like `archlinux-xxx.iso`) and show progress:
 
-`dd if={{path/to/file.iso}} of=/dev/{{usb_drive}}`
+`dd status=progress if={{path/to/file.iso}} of=/dev/{{usb_drive}}`
```

Comment:
This feature is not present on the POSIX specification

Reply:
Reverted in [3adf817](https://github.com/tldr-pages/tldr/pull/12442/commits/3adf8170f53565f78f34576de7a61b1bc31729a0)